app/src/main/res/values/dimens.xml

```diff
@@ -20,5 +20,6 @@
     <!-- fabs -->
     <dimen name="fab_big">56dp</dimen>
     <dimen name="fab_normal">76dp</dimen>
+    <dimen name="text_margin">16dp</dimen>
```

Comment:
misplaced - under the <--- fab --> section.
Probably should be elsewhere... !fix